sdk/tables/azure-data-tables/azure/data/tables/_serialize.py

```diff
@@ -212,10 +212,6 @@ def _add_entity_properties(source):
     # set properties type for types we know if value has no type info.
     # if value has type info, then set the type to value.type
     for name, value in to_send.items():
-        if "@odata.type" in name or name + "@odata.type" in to_send:
```

Comment:
Why revokes this update? There won't have any change in our code in this PR then.